.github/guides/POLICYCONFIG.md

```diff
@@ -0,0 +1,35 @@
+Welcome to this short guide to the POLICY config mechanism.
```

Comment:
I don't believe we have policy at all.

Reply:
Alright, I thought so. I'll remove that one